apps/daedong-user/src/components/ClubPost.tsx

```
@@ -0,0 +1,226 @@
+              <DetailTitle>면접 절차</DetailTitle>
+              <InterviewProcess />
+            </DetailSection>
+
+            <DetailSection>
+              <DetailTitle>과제 정보</DetailTitle>
+              <DetailContent>{homeworkInfo}</DetailContent>
+            </DetailSection>
+
+            <SupportBtnWrapper>
+              <Button children="지원하기" color="white" backgroundHoverColor='#e3746c' />
+            </SupportBtnWrapper>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)